src/sieppari/util.cljc

```diff
@@ -0,0 +1,4 @@
+(ns sieppari.util)
```

Comment:
maybe this can live in `sieppari.async` instead of a new namespace, but haven't checked all the usages and not that big of a deal :smile: